CONTRIBUTING.md

```diff
@@ -1,14 +1,63 @@
 ## Build
 
-Sloop uses GitHub to manager reviews of pull requests
+Sloop uses GitHub to manage reviews of pull requests
+
+[Singing up for a GitHub account](https://docs.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account) if you don't have one
+
+
 
 ## Steps to Contribute
 
-ADD
+1.[Pick an issue you are interested in working on](https://github.com/salesforce/sloop/issues).
+It’s a good idea to comment on an Issue that you want to help with and ask some questions before doing any major coding work
+
+2.Fork the repository: Look for the “Fork” button in the top right corner of [salesforce/sloop repo](https://github.com/salesforce/sloop)
+
+3.Clone the fork: Click on the green “clone or download” button,you can choose “clone with HTTPS.” Or, you can use SSH, which is compatible with 2-factor authentication, but just [some more work to set up](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+Go to the folder where you want to save the repository, type `git clone that-https-url-you-copied`
+
+4: Add a remote pointing to the original repository so you can sync changes: `git remote add upstream git@github.com:salesforce/sloop.git`
+
+5.Make a git branch: If you type git status you will see the branch name that you’re on, called master. In most projects, master is a special place where the most stable, reviewed, up-to-date code is. So, you’ll need to make your own branch: 
+
+`git branch a-descriptive-name`
```

Comment:
we don't need this command, the command below creates and switches to a new branch

Reply:
@nurland I think it's better to keep step5 as two steps: create a new branch as one step and switch to the branch as another step. We definitely can combine them into one command as `git checkout -b name-for-your-branch` but I still think it's more descriptive and clear to keep two steps. FYI @kritikapradhan

Reply:
I think most people use just `git checkout -b` instead of 2 longer commands where you need to type branch name twice. But I don't feel strong about it.

Reply:
Sounds reasonable, already updated.